package.json

```diff
@@ -51,6 +51,7 @@
     "gatsby-plugin-google-tagmanager": "^2.1.25",
     "gatsby-plugin-react-helmet": "^3.1.14",
     "gatsby-plugin-sitemap": "^2.2.27",
+    "memoizerific": "^1.11.3",
```

Comment:
We already had this as a dev-dependency, so I added it to our normal dependencies to use. It is only used at build-time. Was getting an `eslint(import/no-extraneous-dependencies)` warning, so figured this was the proper thing to do.

This package lets us memoize function calls, I was looking this to memoize stitch calls where possible.